src/components/PolicyModal.js

```diff
@@ -8,27 +8,29 @@ const AttendancePolicyModal = () => {
       <Button onClick={() => setVisible(true)}>
         Attendance Policy
       </Button>
-      <Modal
-        show={visible}
-        onClose={() => setVisible(false)}
-      >
-        <Modal.Header>
-          Attendance Policy
-        </Modal.Header>
-        <Modal.Body>
-          <div className="space-y-6">
-            <p>
-              Regular and punctual attendance is an important part of a students education. Class begins promptly and attendance is strictly monitored. Students should not be late. It is disruptive and unprofessional to enter a class that is already underway.
-            </p> 
-            <h1>Tardy</h1>
-            <p>
-              A student is considered tardy if they miss the morning standup
-              With the exception of emergencies, students will notify an instructor by Slack message 24 hours in advance of any absences. 
-              A student is considered absent when they miss one or more hour If three or more absences occur for any reason, the status of the student will be reviewed by the instructor to determine a disposition. If a student reaches three
-            </p>
-          </div>
-        </Modal.Body>
-      </Modal>
+      <div onClick={() => setVisible(false)}>
+        <Modal
+          show={visible}
+          onClose={() => setVisible(false)}
+        >
+          <Modal.Header>
+            Attendance Policy
+          </Modal.Header>
+          <Modal.Body>
+            <div className="h-[60vh] space-y-6 overflow-y-scroll p-6">
+              <p>Regular and punctual attendance is an important part of a student&#39;s education. Class begins promptly and attendance is strictly monitored. Students should not be late. It is disruptive and unprofessional to enter a class that is already underway.</p>
+              <p>With the exception of emergencies, students will notify an instructor by Slack message 24 hours in advance of any tardies or absences.</p>
+              <p>A student is considered <strong>tardy</strong> if:</p>
+              <ul>- they miss the morning check in.</ul>
```

Comment:
I think the ul will provide the syntax for the bullet point. You can remove the `-` from all these lines. That is markdown syntax for bullet points.